server/src/main/java/org/apache/iotdb/db/qp/physical/crud/AlignByDevicePlan.java

```diff
@@ -48,6 +48,7 @@
 
   // to record result measurement columns, e.g. temperature, status, speed
   private List<String> measurements;
+  private Set<String> withoutNullColumns = new HashSet<>();
```

Comment:
add some comments about `withoutNullColumns` which indicating that it contains alias.

Reply:
fix